_data/marynaggita/README.md

```diff
@@ -0,0 +1,36 @@
+# [marynaggita](https://github.com/marynaggita/)'s Zarr Collection
+
+## Tweet Mentions
+
+- https://twitter.com/latlong_blog/status/1543726888908009472
+- https://twitter.com/_jhamman/status/1579882167303168004
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+
+
+## Zarr Data Visualisation
+
+Atmospheric Conditions from the Coupled Model Intercomparison Project Phase 6 (CMIP6)(https://www.wdc-climate.de/ui/cmip6?input=CMIP6.CMIP.AS-RCEC.TaiESM1.1pctCO2) 
+
+
+## Code Snippets
+
+- NASA Prediction of Worldwide Energy Resources (POWER) - https://registry.opendata.aws/nasa-power
```

Comment:
This code has already been posted in the repository. Can you please try to find another code snippet or maybe create one on your own? 😄